From: Dan Carpenter <dan.carpenter@oracle.com>
Date: Thu, 14 Apr 2016 12:37:44 +0300
Subject: hpsa: set the enclosure identifier to zero
Git-commit: aa105695732daa6604cb017ceb59a05ef34956bd
Patch-mainline: v4.7-rc1
References: bsc#964462, FATE#319992

This has only called from show_sas_rphy_enclosure_identifier().  The
caller expects that we set an identifier, otherwise it uses an
uninitialized variable.

[mkp: fixed typo]

Signed-off-by: Dan Carpenter <dan.carpenter@oracle.com>
Acked-by: Don Brace <don.brace@microsemi.com>
Signed-off-by: Martin K. Petersen <martin.petersen@oracle.com>
Acked-by: Johannes Thumshirn <jthumshirn@suse.com>
---
 drivers/scsi/hpsa.c |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/scsi/hpsa.c
+++ b/drivers/scsi/hpsa.c
@@ -9619,6 +9619,7 @@ hpsa_sas_get_linkerrors(struct sas_phy *
 static int
 hpsa_sas_get_enclosure_identifier(struct sas_rphy *rphy, u64 *identifier)
 {
+	*identifier = 0;
 	return 0;
 }
 
